cmd/remove-members/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestReadMemberList(t *testing.T) {
+	memberList, err := readMemberList("test/members.txt")
```

Comment:
Would prefer you move this to `testdata`, it's more idiomatic (see `go help test`)
```suggestion
	memberList, err := readMemberList("testdata/members.txt")
```